core/src/hir/attrs.rs

```diff
@@ -54,11 +54,17 @@ pub struct Attrs {
 #[non_exhaustive]
 #[derive(Clone, Default, Debug)]
 pub struct DemoInputCFG {
-    /// `#[diplomat(input(label = "..."))]`
+    /// `#[diplomat::demo(input(label = "..."))]`
     /// Label that this input parameter should have. Let demo_gen pick a valid name if this is empty.
     ///
     /// For instance <label for="v">Number Here</label><input name="v"/>
     pub label: String,
+
+    /// `#[diplomat::demo(input(default_value = "..."))]`
+    /// Sets the default value for a parameter.
+    ///
+    /// Should ALWAYS be a string. The HTML renderer is expected to do validation for us.
```

Comment:
thought: technically we can support integers here too, and parse them to strings below.